pkg/sync/v6/sync.go

```diff
@@ -275,20 +276,25 @@ var (
 		return o.GroupVersionKind().GroupKind() == schema.GroupKind{Kind: "Secret"} ||
 			o.GroupVersionKind().GroupKind() == schema.GroupKind{Kind: "ConfigMap"}
 	}
+	sharedResourceFilter = func(o unstructured.Unstructured) bool {
+		// if Template OR ImageStream in openshift namespace
+		return o.GetNamespace() == "openshift" &&
+			(o.GroupVersionKind().GroupKind() == schema.GroupKind{Group: "image.openshift.io", Kind: "ImageStream"} ||
```

Comment:
are you sure we don't need `schema.GroupKind{Kind: "ImageStream"} and schema.GroupKind{Kind: "Template"}` as well?

Reply:
all the current object complies with the current one. But need to confirm,